spaceship.zsh

```diff
@@ -70,12 +74,24 @@ spaceship_user() {
 spaceship_host() {
   if [[ -n $SSH_CONNECTION ]]; then
     echo -n "$(spaceship_user)"
-    echo -n " %B${SPACESHIP_PREFIX_USER}%b "
+
```

Comment:
I was wondering about that. I saw that elsewhere in your code, but I couldn't find any documentation supporting it. Thanks.